### PR TITLE
Update Pattern 1 to be compatible with MS SQL

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: pattern-1/bosh-release/config/blobs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+jdbcdrivers/mssql-jdbc-7.0.0.jre8.jar:
```
Comment (VimukthiPerera) [inline pattern-1/bosh-release/config/blobs.yml]: Shall we empty this file as discussed offline?

File: pattern-1/bosh-release/jobs/wso2apim/spec [hunk 2]
```diff
@@ -43,20 +46,23 @@ properties:
   wso2apim.address:
     description: wso2apim address
 
-  wso2apim.carbon_db.hostname:
-    description: WSO2 Carbon database server hostname
-  wso2apim.carbon_db.db_name:
-    description: WSO2 Carbon database database name
+  wso2apim.carbon_db.jdbc_url:
+    description: WSO2 Carbon JDBC URL
```
Comment (VimukthiPerera) [inline pattern-1/bosh-release/jobs/wso2apim/spec]: Shall we change the description to "WSO2 Carbon database JDBC URL"?

File: pattern-1/bosh-release/jobs/wso2apim/spec [hunk 3]
```diff
@@ -43,20 +46,23 @@ properties:
...
   wso2apim.carbon_db.password:
     description: WSO2 Carbon database password
 
-
-  wso2apim.am_db.hostname:
-    description: WSO2 APIM database server hostname
-  wso2apim.am_db.db_name:
-    description: WSO2 APIM database database name
+  wso2apim.am_db.jdbc_url:
+    description: WSO2 APIM JDBC URL
```
Comment (VimukthiPerera) [inline pattern-1/bosh-release/jobs/wso2apim/spec]: Shall we change the description to "WSO2 APIM database JDBC URL"?

File: pattern-1/bosh-release/jobs/wso2apim/templates/repository/conf/registry.xml [hunk 4]
```diff
@@ -1,335 +1,335 @@
...
+    <dbConfig name="wso2registry">
+        <dataSource>jdbc/WSO2CarbonDB</dataSource>
+    </dbConfig>
+
+    <dbConfig name="govregistry">
+        <dataSource>jdbc/WSO2REG_DB</dataSource>
+    </dbConfig>
+    <remoteInstance url="https://publisher.apim-wso2.com">
+        <id>gov</id>
+        <cacheId><%=properties.wso2apim.mysql.username %>@jdbc:mysql://<%=properties.wso2apim.mysql.hostname %>:3306/<%=properties.wso2apim.mysql.product_db %></cacheId>
```
Comment (VimukthiPerera) [inline pattern-1/bosh-release/jobs/wso2apim/templates/repository/conf/registry.xml]: Shall we add the proper parameter here?

File: pattern-1/bosh-release/jobs/wso2apim_analytics/spec [hunk 5]
```diff
@@ -32,32 +32,38 @@ properties:
   wso2apim_analytics.address:
     description: wso2apim address
 
-  wso2apim.carbon_db.hostname:
-    description: WSO2 APIM MySQL server hostname
-  wso2apim.carbon_db.db_name:
-    description: WSO2 APIM MySQL database name
+  wso2apim.carbon_db.jdbc_url:
+    description: WSO2 Carbon JDBC URL
```
Comment (VimukthiPerera) [inline pattern-1/bosh-release/jobs/wso2apim_analytics/spec]: Shall we change the description to "WSO2 Carbon database JDBC URL"?

File: pattern-1/bosh-release/jobs/wso2apim_analytics/spec [hunk 6]
```diff
@@ -32,32 +32,38 @@ properties:
...
     description: WSO2 APIM database username
   wso2apim.am_db.password:
     description: WSO2 APIM database password
 
-  wso2apim.sp_cluster_db.hostname:
-    description: WSO2 APIM Analytics cluster database server hostname
-  wso2apim.sp_cluster_db.db_name:
-    description: WSO2 APIM Analytics cluster database database name
+  wso2apim.sp_cluster_db.jdbc_url:
+    description: WSO2 APIM Analytics JDBC URL
```
Comment (VimukthiPerera) [inline pattern-1/bosh-release/jobs/wso2apim_analytics/spec]: Shall we change the description to "WSO2 APIM Analytics database JDBC URL"?

File: pattern-1/bosh-release/jobs/wso2apim_analytics/spec [hunk 7]
```diff
@@ -32,32 +32,38 @@ properties:
...
   wso2apim.carbon_db.password:
-    description: WSO2 APIM MySQL password
+    description: WSO2 Carbon database password
 
-  wso2apim.am_db.hostname:
-    description: WSO2 APIM database server hostname
-  wso2apim.am_db.db_name:
-    description: WSO2 APIM database database name
+  wso2apim.am_db.jdbc_url:
+    description: WSO2 APIM JDBC URL
```
Comment (VimukthiPerera) [inline pattern-1/bosh-release/jobs/wso2apim_analytics/spec]: Shall we change the description to "WSO2 APIM database JDBC URL"?

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: pattern-1/bosh-release/config/blobs.yml (hunk 1 above)
Comment (Savidude) [reply pattern-1/bosh-release/config/blobs.yml]: fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d

File: pattern-1/bosh-release/jobs/wso2apim/spec (hunk 2 above)
Comment (Savidude) [reply pattern-1/bosh-release/jobs/wso2apim/spec]: fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d

File: pattern-1/bosh-release/jobs/wso2apim/spec (hunk 3 above)
Comment (Savidude) [reply pattern-1/bosh-release/jobs/wso2apim/spec]: fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d

File: pattern-1/bosh-release/jobs/wso2apim/templates/repository/conf/registry.xml (hunk 4 above)
Comment (Savidude) [reply pattern-1/bosh-release/jobs/wso2apim/templates/repository/conf/registry.xml]: fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d

File: pattern-1/bosh-release/jobs/wso2apim_analytics/spec (hunk 5 above)
Comment (Savidude) [reply pattern-1/bosh-release/jobs/wso2apim_analytics/spec]: fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d

File: pattern-1/bosh-release/jobs/wso2apim_analytics/spec (hunk 6 above)
Comment (Savidude) [reply pattern-1/bosh-release/jobs/wso2apim_analytics/spec]: fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d

File: pattern-1/bosh-release/jobs/wso2apim_analytics/spec (hunk 7 above)
Comment (Savidude) [reply pattern-1/bosh-release/jobs/wso2apim_analytics/spec]: fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d

**Review by VimukthiPerera — APPROVED** [review]

**Pull request merged**